app/controllers/sessions.rb

```diff
@@ -0,0 +1,27 @@
+class Chitter < Sinatra::Base
+
+  get '/sessions/new' do
+    redirect('/peeps') if not session[:user_id].nil?
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)<br>Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)